frmts/vrt/vrtderivedrasterband.cpp

```diff
@@ -872,6 +872,18 @@ CPLErr VRTDerivedRasterBand::GetPixelFunctionArguments(
                         dfVal = this->GetScale(&success);
                     else if (osArgName == "offset")
                         dfVal = this->GetOffset(&success);
+                    else if (osArgName == "xoff")
+                        dfVal = static_cast<double>(nXOff);
+                    else if (osArgName == "yoff")
+                        dfVal = static_cast<double>(nYOff);
+                    else if (osArgName == "geotransform")
+                    {
+                        std::array<double, 6> gt;
+                        GetDataset()->GetGeoTransform(gt.data());
```

Comment:
shouldn't we test the return of GetGeoTransform() and error out if it returns CE_Failure ?

Reply:
Updated to test the return but ignore a failure here, see comment